### PR TITLE
Data Science with Python: Getting started with Linear Algebra

### PR DESCRIPTION
**Pull request by Sera001**
<hr>

## Description 📜

A short introduction to get started with Linear Algebra for datascience along with simple python codes for quick mathematical calculation using numpy library.
Issue no: #438

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [x] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done :

Keep your project title as "Getting Started with Linear Algebra"
Your README.md file and .ipynb file needs more elaboration
Follow the project structure given
The project file name should be in snake case : getting started with linear algebra.ipynb
If you had referred any resources add them up in "DS Resources" folder

Kindly work on it. @Sera001

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done :

The file name of .ipynb file should be in snake case : getting_started_with_linear_algebra.ipynb
Your main README.md needs more detailing.

Kindly work on it. @Sera001

**Comment by prathimacode-hub** [comment]
Have you done the changes? @Sera001

**Comment by Sera001** [comment]
Yes, I did......
image

File: Datascience_With_Python/Linear Algebra/Getting started with Linear Algebra/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          2.4 TRANSPOSE MATRIX
+          2.5 INVERSE OF A MATRIX
+>**3. EIGEN VECTORS AND VALUES**
+
+## Applications:
+   1. Plays a major role in managing and simplifing the huge datasets required for datascience.
+   2. Linear regression can be easily found with the help of Linear algebra.
+   3. Natural Language Processing also requires Linear algebra for managing the hugwe amount of words.
+   4. Linear Algebra has a importance in Computer Vision for image processing techniques.
+
```
Comment (DhruvBajaj01) [inline Datascience_With_Python/Linear Algebra/Getting started with Linear Algebra/README.md:31]: Add python libraries that are being used for the demonstration of Linear Algebra.

**Review by DhruvBajaj01 — COMMENTED** [review]
Please go through the reviews by other mentors and also add libraries that are being used in the readme.

**Review by DhruvBajaj01 — COMMENTED** [review]
Great work! Reviewed.

**Comment by DhruvBajaj01** [comment]
@prathimacode-hub  please give it a final review and merge the file.

**Comment by Sera001** [comment]
Thanks a lot!! @DhruvBajaj01

**Review by prathimacode-hub — APPROVED** [review]
PR approved. 👍@Sera001

**Review by DhruvBajaj01 — COMMENTED** [review]
GREAT WORK @Sera001! keep contributing.

**Comment by Sera001** [comment]
Thanks a lot for the suggestions and support! @DhruvBajaj01 and @prathimacode-hub

**Pull request merged**